src/refactor_nrepl/find/find_symbol.clj

```diff
@@ -137,7 +137,16 @@
                                (str/join "/" [namespace var-name]))]
     (->> (core/dirs-on-classpath)
          (mapcat (partial core/find-in-dir (some-fn core/clj-file? core/cljc-file?)))
-         (mapcat (partial find-symbol-in-file fully-qualified-name ignore-errors)))))
+         (shuffle) ;; make it less likely that work will be concentrated in one partition (see next line)
+         (find-util/divide-by (find-util/processor-count))
+         (pmap (fn [work]
+                 ;; `vec` ensures all work is actually performed within the surrounding `pmap`
+                 (->> work
+                      (mapcat (comp vec
+                                    (partial find-symbol-in-file fully-qualified-name ignore-errors)))
+                      vec)))
```

Comment:
Btw I think `doall` signals much better that the intention is to realize a lazy value.

use of `vec` is likely to get refactored away when future selves or other developers don't see the point of conversion to this data type.

This can be changed in a few places in this diff.